scripts/sort-issues.py

```diff
@@ -0,0 +1,86 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: py:hydrogen
+#     text_representation:
+#       extension: .py
+#       format_name: hydrogen
+#       format_version: '1.3'
+#       jupytext_version: 1.10.3
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
```

Comment:
ahhhh... prototyping in the notebook 😉

Reply:
indeed - it means you can open up this python script *as a notebook* with jupytext, which is helpful for modifications and real-time prototyping